todoTask/todoTask.js

```diff
@@ -1,85 +1,59 @@
-﻿// ------------------------global varibals--------------------
-let globalID = 0;
-// -------------------------functions---------------------------
-const getId = () => (globalID += 1);
+const getId = (() => {
+    let globalID = 0;
+    return () => (globalID += 1);
+})();
 
-const makeTodo = (title, complete) => {
-    return {
-        id: getId(),
-        title,
-        complete,
-    };
-};
+const createTodo = (title, complete = false) => ({
+    id: getId(),
+    title,
+    complete,
+});
 
-const makeTodos = (n) => {
-    let todos = [];
-    for (let i = 1; i <= n; i++) {
-        let title = `title${i}`;
-        let complete = false;
-        let todo = makeTodo(title, complete);
-        todos = addTodo(todos, todo);
-    }
-    return todos;
-};
+const createTodos = (count) =>
```

Comment:
`n` is not very clear name.
In general one-char variables are not recommended. The only exception usually is `i` for index